src/content/api/cli.md

```diff
@@ -314,7 +314,7 @@ Parameter         | Explanation                              | Usage
 Shortcut | Replaces
 ---------|----------------------------
 -d       | `--debug --devtool cheap-module-eval-source-map --output-pathinfo`
--p       | `--optimize-minimize --define process.env.NODE_ENV="production"`, see [building for production](/guides/production)
+-p       | `--mode production`, see [building for production](/guides/production/)
```

Comment:
If this is helpful, In `webpack-cli v4` beta, we have `--prod` flag for this.

Reply:
Does that mean `-p` is the same as `--prod` in webpack-cli 4+ ?

Reply:
Yes, They mean same.
For reference, @EugeneHlushko here is the whole list of commands and flags and their alias from `webpack-cli v4 beta`.
![image](https://user-images.githubusercontent.com/15812317/70707522-56b20680-1cfe-11ea-9f7e-a932e8fe8bed.png)

Reply:
Allright, we will have to update this whole list when its out of beta